pyramid/file.go

```diff
@@ -7,36 +7,16 @@ import (
 
 // File is pyramid wrapper for os.file that triggers pyramid hooks for file actions.
 type File struct {
-	fh *os.File
+	*os.File
 
 	closed    bool
 	persisted bool
 	store     func(string) error
 }
 
-func (f *File) Read(p []byte) (n int, err error) {
-	return f.fh.Read(p)
-}
-
-func (f *File) ReadAt(p []byte, off int64) (n int, err error) {
-	return f.fh.ReadAt(p, off)
-}
-
-func (f *File) Write(p []byte) (n int, err error) {
-	return f.fh.Write(p)
-}
-
-func (f *File) Stat() (os.FileInfo, error) {
-	return f.fh.Stat()
-}
-
-func (f *File) Sync() error {
-	return f.fh.Sync()
-}
-
 func (f *File) Close() error {
```

Comment:
This existence of this function makes me doubt that the change at line 10 is safe.  Specifically, it makes `pyramid.File` have a safe conversion to `File`.  So now I can
```go
func logAndClose(f *os.File) {
  log("closing!")
  f.Close()
}

func unsafe(f *pyramid.File) {
  // use f...
  logAndClose(f.File)
}

Reply:
Good catch, fixed!